client/app/scripts/charts/__tests__/nodes-layout-test.js

```diff
@@ -131,6 +131,18 @@ describe('NodesLayout', () => {
       edges: fromJS({
         [edge('n1', 'n4')]: {id: edge('n1', 'n4'), source: 'n1', target: 'n4'}
       })
+    },
+    layoutProps: {
+      nodes: fromJS({
+        n1: {id: 'n1', label: 'lold', labelMinor: 'lmold', rank: 'rold'},
```

Comment:
Shouldn't we also set `x`, `y` values here, like we do with `rank`, to make sure they are not overrwritten?